src/card.tsx

```diff
@@ -21,7 +21,6 @@ export const Card = forwardRef<
     className={clsx(
       'rounded-xl',
       'min-w-52',
-      theme === 'yellow' && 'bg-yellow-200 border border-yellow-300',
```

Comment:
is there any value to leaving this as a theme option but making gray the default?

Reply:
I don't really think so? It's trivial to put back if needed, and if not needed, it's just another case/conditional branch that needs to be maintained and tested.